docs/products/flink/howto/manage-credentials-jars.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Credential management for JAR applications
+---
+
+Learn how to use the `AVN_CREDENTIALS_DIR` environment variable to securely manage credentials for custom JARs in Aiven for Apache Flink®.
+
+## About credential management
+
+[Custom JARs](/docs/products/flink/concepts/custom-jars) in Aiven for Apache Flink®
+enable you to connect your Apache Flink jobs with Aiven's supported connectors and any
+external systems you manage, which is essential for real-time stream processing.
+Effectively managing credentials is critical to securing and ensuring compliant
+access to these services.
+
+Aiven for Apache Flink® securely manages credentials for your JAR applications using
+the `AVN_CREDENTIALS_DIR` environment variable. This centralizes credentials for internal
+and external integrations, ensuring policy-compliant and secure access
+to sensitive information.
+
+## Prerequisites
+
+- An active Aiven for Apache Flink service
+- [Integration with services: Aiven for Apache Kafka or PostgreSQL](/docs/products/flink/howto/create-integration)
+ or external Apache kafka service
+- Permission to create a [JAR application for Aiven for Apache Flink](/docs/products/flink/howto/create-jar-application)
+
+## Credential provisioning
+
+Aiven streamlines credential management for Aiven-managed and external services using
+the `AVN_CREDENTIALS_DIR` environment variable. This variable points to a
+directory with integrated service credentials, providing access for users while
+abstracting the intricacies of internal storage.
+
+:::note
+You can have multiple credentials for multiple Apache Kafka services in
+the same `AVN_CREDENTIALS_DIR` directory.
+:::
+
+Aiven automatically generates credentials for integrated services and stores them in JSON
+files named by each service's `integration_id`. For example, credentials for an Apache
+Kafka service with an `integration_id` such as `my_kafka_service` is stored in a
+file named `my_kafka_service.json`. This file is in a directory your
+Aiven for Apache Flink® application can access. The path to this folder
+is `/AVN_CREDENTIALS_DIR/my_kafka_service.json`.
+
+## Access credentials in JAR applications
+
+1. Locate the credentials file
+
+   1. Identify the `integration_id` of your service from the [integration list](/docs/tools/cli/service/integration#avn_service_integration_list).
+   1. Retrieve the corresponding credentials file named `{integration_id}.json`
+   located at `/AVN_CREDENTIALS_DIR/`. For example, if your service's `integration_id`
```

Comment:
Still unsure if I can re-review, or if you're done, but the indentation must be fixed through the files of this PR, please 🙂 This is not a comment on _this_ line, specifically. 

every time you see a bullet like this 

```
1. some words
and a new line
1. some more words
```

we want to have instead:

```
1. some words
   and a new line
1. some more words
```

so that the text start everywhere the same place.

Reply:
Done! Thanks!